static/docs/commands-reference/status.md

```diff
@@ -48,22 +48,47 @@ detected, `dvc status` prints this message:
     Pipeline is up to date. Nothing to reproduce.
 ```
 
-This says that no differences were detected, and therefore that no stages would
-be run again if `dvc repro` were executed.
+This indicates that no differences were detected, and therefore no stages would
+be run again by `dvc repro`.
 
 If instead, differences are detected, `dvc status` lists those changes. For each
-DVC-file (stage) with differences, the _dependencies_ and/or _outputs_ that
-differ are listed. For each item listed, either the file name or the checksum is
-shown, and additionally a status word is shown describing the change:
+DVC-file (stage) with differences, the changes in _dependencies_ and/or
+_outputs_ that differ are listed. For each item listed, either the file name or
+the checksum is shown, and additionally a status word is shown describing the
+changes. This changes list provides a reference to both the status of a
+DVC-file, as well as the changes to individual dependencies and outputs
+described in it:
 
 - For the local workspace:
```

Comment:
This shouldn't be a bullet with sub-bullets. How about using a `###` header? Or just a **bold** phrase.

Reply:
...and un-indent the bullets below, of course 🙂